orchagent/qosorch.cpp

```diff
@@ -1898,7 +1892,7 @@ void QosOrch::doTask(Consumer &consumer)
             continue;
         }
 
-        auto task_status = (this->*(m_qos_handler_map[qos_map_type_name]))(consumer);
+        auto task_status = (this->*(m_qos_handler_map[qos_map_type_name]))(consumer, it->second);
```

Comment:
Do we need to pass consumer any more?

Reply:
@neethajohn  
Yes, we do.
`doTask` => `handleDscpToTcTable` => `processWorkItem`
`processWorkItem` is shared for many QoS map handlers. In this function, `consumer` is required for fetching table name which is required `m_qos_maps`.